src/applications/debt-letters/const/deduction-codes/index.js

```diff
@@ -160,7 +160,7 @@ export const renderWhyMightIHaveThisDebt = deductionCode => {
               status.
             </li>
             <li>
-              You‘ve received two payments for the same compensation and pension
+              You’ve received two payments for the same compensation and pension
```

Comment:
@bradl3yC @craigwheeler This is the only file I touched that belongs to your team. Thanks!